examples/Widget Basics.ipynb

```diff
@@ -196,7 +196,7 @@
    "source": [
     "Widgets are represented in the back-end by a single object.  Each time a widget is displayed, a new representation of that same object is created in the front-end.  These representations are called views.\n",
     "\n",
-    "![Kernel & front-end diagram](images/WidgetModelView.png)"
```

Comment:
It's just an API rename, spawned from a discussion between @SylvainCorlay and I.  The PR is not actually meant to merged.